lib/platform/bitbucket-server/index.ts

```diff
@@ -549,6 +552,30 @@ export function addAssignees(iid: number, assignees: string[]): Promise<void> {
   return Promise.resolve();
 }
 
+function isInvalidReviewersResponse(err: any): boolean {
+  const errors = err?.response?.body?.errors || [];
+  return (
+    errors.length > 0 &&
+    errors.every(
+      (error) => error?.exceptionName === BITBUCKET_INVALID_REVIEWERS
+    )
+  );
+}
+
+function getInvalidReviewers(err: any): string[] {
+  const errors = err?.response?.body?.errors || [];
+  let invalidReviewers = [];
+  for (const error of errors) {
+    if (error.exceptionName === BITBUCKET_INVALID_REVIEWERS) {
+      invalidReviewers = invalidReviewers.concat(
+        error.reviewerErrors?.map(({ context }) => context) || []
+      );
+    }
+  }
+
+  return invalidReviewers;
+}
+
```

Comment:
Lets move this helper to a seperate file (eg `utils.ts` or `re), as this file is already very big.

Reply:
Will do

Reply:
I moved them and also added proper types for the Error.